docs/SceneWalkthrough/StoreWalkthrough.md

```diff
@@ -0,0 +1,8 @@
+## **Store Demo**
+This demo showcases the store interface through a faux store.
+- There are two options, ``CaliforniumGeneralAudience``, and ``DevCaliforniumAudience``.
+- The ``Checkout`` buttons to the right open up the epic store check out page with the respective product in the cart
+- ``Query Offers`` refreshes the available items in the shop.
+
```

Comment:
ive never actually seen this work, but im assuming something needs to be set up on the dev portal, could use a note and link on that

Reply:
I was looking for documentation on it, but there's basically none on how to actually use the dev portal, I only could find the SDK documentation

Reply:
yeah i cant really find anything either, theres a few docs like all over the place, i wonder if maybe the bottom of every doc here should say something like
'for more information on the specific features, you can check out the Epic docs here'
https://dev.epicgames.com/docs
that way we can kinda just blatantly wash our hands of any weird feature things that arent connected to the samples?